jaeger-tracing/docker-compose.yaml

```diff
@@ -40,4 +40,4 @@ services:
     environment:
     - COLLECTOR_ZIPKIN_HOST_PORT=9411
     ports:
-    - "${PORT_UI:-10000}:16686"
+    - "${PORT_UI:-16686}:16686"
```

Comment:
this is correct as its not being proxied so makes more sense to expose the default port in this case.

it would also ensure it matches what is written in the docs